test/pbelutest.c

```diff
@@ -46,7 +46,7 @@ static int test_pbelu(void)
         last_type = pbe_type;
         last_nid = pbe_nid;
     }
-    return failed ? 0 : 1;
+    return 0;
```

Comment:
Yes, the assignment is redundant.  I've reworked the function to simplify it a bit.